### ISSUE TITLE
On some older GTK versions, the gscreenshot UI fails to load with an undefined property error

### ISSUE THREAD

**Issue by thenaterhood** [thenaterhood/gscreenshot#75]

**Comment by onetwopunch** [comment]
Not sure if this is what you're referring to, but I'm trying to run gscreenshot in i3 on Ubuntu 16.04 and when running the GUI, I get this error:
```
GLib.Error: gtk-builder-error-quark: .:87:51 Invalid property: GtkScrolledWindow.propagate_natural_width (11)

(gscreenshot:23826): GLib-GObject-WARNING **: instance with invalid (NULL) class pointer

(gscreenshot:23826): GLib-GObject-CRITICAL **: g_signal_handlers_disconnect_matched: assertion 'G_TYPE_CHECK_INSTANCE (instance)' failed

(gscreenshot:23826): GLib-GObject-WARNING **: instance with invalid (NULL) class pointer

(gscreenshot:23826): GLib-GObject-CRITICAL **: g_signal_handlers_disconnect_matched: assertion 'G_TYPE_CHECK_INSTANCE (instance)' failed

```

**Comment by thenaterhood** [comment]
That looks familiar. I'll have to check what the error is that I get on mine.

**Comment by thenaterhood** [comment]
Should be fixed in 2.12.0
